.gitlab/ci/.gitlab-ci.on-push.yml

```
@@ -781,19 +790,22 @@ test-upload-flow:
   extends:
     - .default-job-settings
   script:
-    - ./Tests/scripts/test_playbooks_results.sh
+    - ./Tests/scripts/test_end_to_end_results.sh
```

```suggestion
    - ./Tests/scripts/test_playbooks_results.sh
    - ./Tests/scripts/test_end_to_end_results.sh
```

revert after testing